### ISSUE TITLE
Accessibility Issues

### ISSUE THREAD

**Issue by AdamP92** [kieffer-reitenbach/web-design-2-independent-project#9]
Not many accessibility issues, maybe could use an alt attribute or aria details description of the images so people can still understand what the images are if they fail to load.
